app/hooks/use_external_link.ts

```diff
@@ -47,7 +47,7 @@ export function useExternalLink(
             ...overwriteQueryParams,
             ...existingQueryParamsObj,
         };
-        parsedUrl.search = new URLSearchParams(queryParams).toString();
+        parsedUrl.search = Object.entries(queryParams).map(([key, value]) => `${encodeURIComponent(key)}=${encodeURIComponent(value)}`).join('&');
 
```

Comment:
No new test is needed for this, right?  Since there's probably an existing test that is covering this.

Reply:
I added a new test for what this is trying to fix (%20 being converted into +). My guess (and bad memory) is that any other instance of this behavior should be addressed by the rest of the tests.